relay/channel/task/jsplugin/adaptor.go

```
@@ -136,6 +138,15 @@ func (a *TaskAdaptor) EstimateBilling(c *gin.Context, info *relaycommon.RelayInf
 }
 
 func (a *TaskAdaptor) EstimateBillingValidated(c *gin.Context, info *relaycommon.RelayInfo) (map[string]float64, error) {
+	// 视频按秒定价:模型配置了视频价格表时,系数由网关按请求 + 配置推导
+	// (分辨率 × 正常价/错峰价,锚点 = 最高正常价),不走插件的 billing_ratios 钩子。
+	if _, ok := ratio_setting.GetVideoPrice(info.OriginModelName); ok {
+		taskReq, err := relaycommon.GetTaskRequest(c)
+		if err != nil {
+			return nil, nil
```

golangci-lint (2.13.2)

[error] error is not nil (line 144) but it returns nil

(nilerr)